custom_components/tahoma/sensor.py

```diff
@@ -301,6 +301,11 @@
         native_unit_of_measurement=TIME_SECONDS,
         entity_registry_enabled_default=False,
     ),
+    OverkizSensorDescription(
+        key="core:Memorized1PositionState",
+        name="MY position",
```

Comment:
Why not 'My Position'? Is MY how Somfy writes it in English, in Dutch they just use My.
And we capitalize all words in sensors for now, maybe we should do that differently, but would be good to have them all the same.

Reply:
When I look for it, I always find it in upper case: https://service.somfy.com/downloads/fr_v4/programmation_des_fins_de_courses_vr.pdf

You suggest `My Position`?